src/ucp/core/ucp_request.h

```diff
@@ -36,6 +36,7 @@ enum {
     UCP_REQUEST_FLAG_RNDV                 = UCS_BIT(9),
     UCP_REQUEST_FLAG_OFFLOADED            = UCS_BIT(10),
     UCP_REQUEST_FLAG_BLOCK_OFFLOAD        = UCS_BIT(11),
+    UCP_REQUEST_FLAG_STREAM_RECV          = UCS_BIT(12),
```

Comment:
where is it set?

Reply:
Of course, in `ucp_stream_recv_nb` function which is not implemented in master yet. I can't add this change to  #1986 because this is API change. I'm preparing next PR which merge all stream building block and add testing.

Reply:
It does not look like API change - it is internal flag, which is not visible to the user.

Reply:
It's used in `ucp_stream_recv_request_test` implementation, but the new function call is meaningless until `ucp_stream_recv_np` is not implemented.

Reply:
lets don't create deadlock between PRs..

Reply:
ok, i see

Reply:
why not just remove the assertion for now? seems like it would always fail the assertion anyway, since no one is setting the flag..

Reply:
Why not just keep it as is because

> It's used in `ucp_stream_recv_request_test` implementation, but the new function call is **meaningless** until ucp_stream_recv_np is **not implemented**.

So, this function **must** always fail.

> lets don't create deadlock between PRs..

and don't complicate the upstream process, this change is pretty obvious.

Reply:
ok